FloripaPublicTransportationTests/RestApiTests.swift

```diff
@@ -0,0 +1,343 @@
+//
+//  RestApiTests.swift
+//  FloripaPublicTransportation
+//
+//  Created by Marcelo Gobetti on 2/23/16.
+//  Copyright © 2016 Marcelo Gobetti. All rights reserved.
+//
+
+import XCTest
+@testable import FloripaPublicTransportation
+import OHHTTPStubs
+
+class RestApiTests: XCTestCase, ExpectationProtocol {
```

Comment:
if I run the tests all together I see the simulator closing between each test, also I see some tests failing, can you explain why?